login/index.html

```diff
@@ -39,6 +39,63 @@
             </div>
         </div>
     </header>
+
+    <main>
+        <div class="columnas__login">
+            <div class="login--img">
+                <img src="../img/img-login-registro.png" alt="login_img" title="login_img">
+            </div>
+            <div class="formulario--box">
+                <div class="centrado__loginh2">
+                    <h2>¡Bienvenido de regreso!</h2>
+                    <img class="vector_login--tamaño" src="../img/Vector_login.svg" alt="Vector_login" title="vector_login">
+                </div>
+                <div class="datos">
+                    <p>Correo electrónico</p>
+                    <div class="cuadro__datos">
+                        <div class="correo__img">
+                            <img class="correo__img--tamaño"src="../img/mail_login.svg" alt="mail" title="mail">
+                        </div>
+                        <div>
+                            <input class="datos__input" type="email" name="correo" id="correo" placeholder="correo electrónico">
+                        </div>
+                        <div class="correo__relleno">
+                            <img class="correo__relleno--tamaño"src="../img/check__login.svg" alt="check__login" title="check__login">
+                        </div>
+                    </div>                    
+                    <p class="contraseña__p">Contraseña</p>
+                    <div class="cuadro__datos">
+                        <div class="contraseña__img">
+                            <img  class="contraseña__img--tamaño" src="../img/Icon-Lock__contraseña.svg" alt="Icon-Lock__contraseña" title="icon-lock__contraseña">
+                        </div>
+                        <div>
+                            <input class="datos__input--pass" type="password" name="contraseña" id="contraseña" placeholder="··············">
+                        </div>
+                        <div class="contraseña__relleno">
+                            <button class="contraseña__relleno--tamaño" onclick="MostrarContraseña()"></button>
```

Comment:
```suggestion
                            <button class="contraseña__relleno--tamaño" id="btn-mostrarcontrasena"></button>
```
onclick luego tiene problemas de ejecución, usa mejor un ID y en JavaScript lo ejecutas mediante eventos